data/pools/transactionPool.go

```diff
@@ -738,15 +739,21 @@ func (pool *TransactionPool) recomputeBlockEvaluator(committedTxIds map[transact
 				pool.statusCache.put(tx, err.Error())
 			}
 
-			switch err.(type) {
+			switch err := err.(type) {
```

Comment:
I think this self-shadowing is weird. I'd rather see `terr :=` or `err2 :=` or `xerr :=` or something.

Reply:
Agreed. Made the change